variables.tf

```
@@ -266,3 +266,9 @@ variable "include_global_resource_types" {
   type        = bool
   default     = true
 }
+
+variable "config_sns_topic_arn" {
+  description = "An SNS topic to stream configuration changes and notifications to."
+  type        = "string"
```

0.11 syntax that 0.12 warns on
```suggestion
  type        = string
```